### ISSUE TITLE
Workflow for reloading

### ISSUE THREAD

**Issue by purcell** [chrisdone/flycheck-stack#5]
Once the backend process is running, if the user adds libraries to the cabal file, what should be their workflow such that those changes are picked up by `flycheck-stack`? It seems like `flycheck-stack-destroy` mostly does the trick, but perhaps isn't obvious, and there might be a more efficient and/or automatic way to deal with this case.

**Comment by chrisdone** [comment]
Just killing it does the trick. Although I'm continuing development in this effort over at https://github.com/chrisdone/intero. 👍

**Comment by purcell** [comment]
Interesting, thanks for the pointer.

`intero` looks interesting, along the lines of slime's swank backend, or Cider for Clojure. I feel like there'll still be a case for `flycheck-stack` as a low-overhead flychecking solution.

**Comment by chrisdone** [comment]
Are you interested in taking it over? As far as a feature-set goes it's pretty much done. Although hopefully my upstream changes to flycheck will be merged and then we can see errors for multiple files: https://github.com/flycheck/flycheck/pull/972

**Comment by purcell** [comment]
Yes, possibly. I'd probably want to use it for longer before committing to that, but if you want to add me as a contributor, that'd be fine. The errors-in-multiple-files change looks orthogonal. I guess you want to cover the case where either the current file's code breaks code elsewhere, or where other build errors wrongly make it appear that there are no errors in the current file?

**Comment by chrisdone** [comment]
> Yes, possibly. I'd probably want to use it for longer before committing to that, but if you want to add me as a contributor, that'd be fine. 

Fair enough, I'll do that.

> where other build errors wrongly make it appear that there are no errors in the current file

This case. More background here: https://github.com/flycheck/flycheck/issues/966

In the end you just change [this](https://github.com/chrisdone/flycheck-stack/blob/master/flycheck-stack.el#L221-L226) to [this](https://github.com/chrisdone/intero/blob/master/elisp/intero.el#L274-L282) and the upstream flycheck change will work.
